packages/yoastseo/src/languageProcessing/helpers/sentence/sentencesLength.js

```diff
@@ -1,28 +1,30 @@
 import wordCount from "../word/countWords.js";
-import { forEach } from "lodash-es";
+import {filter, forEach} from "lodash-es";
 import { stripFullTags as stripHTMLTags } from "../sanitize/stripHTMLTags.js";
 
 /**
  * Returns an array with the number of words in a sentence.
  *
  * @param {Array} sentences Array with sentences from text.
+ * @param {Researcher} 	researcher 	The researcher to use for analysis.
  *
  * @returns {Array} Array with amount of words in each sentence.
  */
-export default function( sentences ) {
+export default function( sentences, researcher ) {
 	const sentencesWordCount = [];
 	forEach( sentences, function( sentence ) {
 		// For counting words we want to omit the HTMLtags.
 		const strippedSentence = stripHTMLTags( sentence );
-		const length = wordCount( strippedSentence );
-
+		// A helper to count characters for languages that don't count number of words for text length.
+		const countCharacters = researcher.getHelper( "customCountLength" );
+		const length = countCharacters ? countCharacters( strippedSentence ) : wordCount( strippedSentence );
 		if ( length <= 0 ) {
 			return;
 		}
 
 		sentencesWordCount.push( {
 			sentence: strippedSentence,
-			sentenceLength: wordCount( strippedSentence ),
+			countLength: length,
```

Comment:
While this is in line with the paragraph length calculation, I don't think you should change the variable name here, or you will have to change that in `packages/yoastseo/src/scoring/helpers/assessments/checkForTooLongSentences.js` and `packages/yoastseo/spec/languageProcessing/researches/countSentencesFromTextSpec.js` as well.

Reply:
@hannaw93 Since you changed the variable name here from `sentenceLength` to `countLength`, you also need to change it in `packages/yoastseo/src/scoring/helpers/assessments/checkForTooLongSentences.js` as @mhkuu mentioned above. On top of that, the variable name in these two spec files also need to be adjusted to `countLength`:
`checkForTooLongSentencesSpec.js` and `SentenceLengthInTextAssessmentSpec.js`

Reply:
@FAMarfuaty I changed it back to `sentenceLength` though :)